pyproject.toml

```diff
@@ -101,3 +102,21 @@ required-imports = ["from __future__ import annotations"]
 files = ["src", "tests"]
 allow_redefinition = true
 ignore_missing_imports = true
+
+[tool.bumpversion]
+current_version = "0.9.4"
+
+[[tool.bumpversion.files]]
+filename = "src/gemdat/__init__.py"
+search = "__version__ = '{current_version}'"
```

Comment:
I think you can leave out the search and replace parts, they should be default. I did a similar PR in the Python template where it works without this, ymmv. https://github.com/NLeSC/python-template/pull/365

Reply:
Ah, I did not realize that. It seems to default to a simple string replacement. One issue I ran into before was when my package version matched the CFF version, both were updated.

Reply:
Ah yeah that's tricky... maybe indeed better to keep it explicit then.